src/Console/ConsoleInput.php

```diff
@@ -77,7 +77,8 @@ public function read()
     public function dataAvailable($timeout = 0)
     {
         $readFds = [$this->_input];
-        $readyFds = stream_select($readFds, null, null, $timeout);
+        $null = null;
```

Comment:
What is the value of this? The repeating null is acceptable in the case it was here.

Reply:
Please revert this unnecessary change.

Reply:
stream_select() expect reference value. And null can't be passed by a reference.

Reply:
PHP doesn't complain when passing `null` for those argument so it's a valid value.

Reply:
@ADmad, so I need to revert this changes?